ts/features/wallet/onboarding/privative/analytics/index.ts

```diff
@@ -0,0 +1,65 @@
+import { getType } from "typesafe-actions";
+import { PrivativeServices } from "../../../../../../definitions/pagopa/privative/configuration/PrivativeServices";
+import { mixpanel } from "../../../../../mixpanel";
+import { Action } from "../../../../../store/actions/types";
+import { getNetworkErrorMessage } from "../../../../../utils/errors";
+import { trackCobadgeResponse } from "../../cobadge/analytics";
+import {
+  addPrivativeToWallet,
+  loadPrivativeIssuers,
+  searchUserPrivative,
+  walletAddPrivativeBack,
+  walletAddPrivativeCancel,
+  walletAddPrivativeChooseIssuer,
+  walletAddPrivativeCompleted,
+  walletAddPrivativeInsertCardNumber,
+  walletAddPrivativeStart
+} from "../store/actions";
+
+export const trackPrivativeAction = (mp: NonNullable<typeof mixpanel>) => (
+  action: Action
+): Promise<any> => {
```

Comment:
```suggestion
): Promise<void> => {
```
Since all these promises return void